docs/examples/sierpinski.html

```diff
@@ -0,0 +1,158 @@
+            hover: false
+          });
+        }
+        render() {
+          var props = this.props;
+          var s = props.size * 1.3;
+          var style = Object.assign({}, dotStyle, {
+            width: s + 'px',
+            height: s + 'px',
+            left: (props.x) + 'px',
+            top: (props.y) + 'px',
+            borderRadius: (s / 2) + 'px',
+            lineHeight: (s) + 'px',
+            background: this.state.hover ? '#ff0' : dotStyle.background
+          });
+          return (
+            h('div', {style: style, onMouseEnter: ()=>this.enter(), onMouseLeave: ()=>this.leave()},
+              this.state.hover ? '*' + props.text + '*' : props.text
+            )
+          );
+        }
+      }
+
+      function SierpinskiTriangle({ x, y, s, children }) {
+        if (s <= targetSize) {
+          return (
+            h(Dot, {x: x - (targetSize / 2), y: y - (targetSize / 2), size: targetSize, text: children})
+          );
+          return r;
+        }
+        var newSize = s / 2;
+        var slowDown = false;
+        if (slowDown) {
+          var e = performance.now()+0.8
+          while (performance.now() < e) {
+            // Artificially long execution time.
+          }
+        }
+
+        s /= 2;
+
+        return [
+          h(SierpinskiTriangle, {x: x, y: y-(s/2), s: s}, children),
+          h(SierpinskiTriangle, {x: x-s, y: y+(s/2), s: s}, children),
+          h(SierpinskiTriangle, {x: x+s, y: y+(s/2), s: s}, children)
+        ];
+      }
+      SierpinskiTriangle.shouldComponentUpdate = function(oldProps, newProps) {
+        var o = oldProps;
+        var n = newProps;
+        return !(
+          o.x === n.x &&
+          o.y === n.y &&
+          o.s === n.s &&
+          o.children === n.children
+        );
+      };
+
+      class ExampleApplication extends dio.Component {
+        constructor() {
+          super();
+          this.state = { seconds: 0 };
+          this.tick = this.tick.bind(this);
+        }
+        componentDidMount() {
+          this.intervalID = setInterval(this.tick, 1000);
+        }
+        tick() {
+          this.setState(state => ({ seconds: (state.seconds % 10) + 1 }))
+        }
+        componentWillUnmount() {
+          clearInterval(this.intervalID);
+        }
+        render() {
+          const seconds = this.state.seconds;
+          const elapsed = this.props.elapsed;
+          const t = (elapsed / 200) % 10;
+          const scale = 1 + (t > 5 ? 10 - t : t) / 10;
+          const transform = 'scaleX(' + (scale / 2.1) + ') scaleY(0.7) translateZ(0.1px)';
+          return (
+            h('div', {style: Object.assign({}, containerStyle, {transform: transform})},
+              h('div',
+                h(SierpinskiTriangle, {x: 0, y: 0, s: 1000},
+                  this.state.seconds
+                )
+              )
+            )
+          );
+        }
+      }
+
+      var start = new Date().getTime();
+      function update() {
```

Comment:
`requestAnimationFrame()` is passed a timestamp as the first parameter (DOMHighResTimeStamp)
https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame